src/entity/Session.ts

```diff
@@ -0,0 +1,17 @@
+import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { Action } from './Action';
+
+@Entity()
+export class Session {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@Column()
+	hashed_uuid: string;
+
+	@Column()
+	ended_at: Date;
```

Comment:
should probably be renamed to end_time or end (ended_at doesnt sound right since this date can be in the future)

Reply:
How is it going to be in the future?

Reply:
Once a token gets refreshed the end date will be set to now + 5 minutes

Reply:
Gotcha makes sense